lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -478,50 +484,40 @@
     margin: ($baseline/2) 0;
     position: relative;
 
-      label {
-        border-bottom: 1px dotted $base-font-color;
-      }
-
```

Comment:
The `label` wraps the form field and the hint, which now appears on focus and hover. There isn't any need for visual indication of an available hint since it will appear by default.